frontend/src/components/GddSend/TransactionForm.spec.js

```diff
@@ -1,5 +1,5 @@
 import { mount } from '@vue/test-utils'
-import TransactionForm from './TransactionForm'
+import TransactionForm from './TransactionForm.vue'
```

Comment:
Is the .vue necessary? I am not sure but if I recall it right we don't use it every where else.
```suggestion
import TransactionForm from './TransactionForm'
 ```